app/Http/Controllers/PlaceController.php

```diff
@@ -85,7 +95,22 @@ public function getAffiliate($id)
 
         return $data;
     }
+
+    public function getRecommend($id){
+
+        # Retrieve the sub-images from the Place_image table.
+        $sub_imgs = DB::table('place_images')->where('place_id', $id)->get();
+
+        # Retrive the recommendation
+        $recommend_places = $this->getRecommendPlace($id);
+        $affiliates = $this->getAffiliate($id);
+        
+        return view('users.place_details.show')
+        ->with('place', $place)
```

Comment:
also this one, we can use array to return data's

Reply:
I didn't put this method. And then I commented out, so it's not used anywhere else.